### ISSUE TITLE
About Curriculum learning 

### ISSUE THREAD

**Issue by ereday** [wojzaremba/algorithm-learning#5]
Hi, 

In Section 5, it is said that :

> For each task, we train a separate model, starting with sequences of complexity 6 and incrementing
by 4 once it achieves 100% accuracy on held-out examples of the current length. Training stops
once the model successfully generalizes to examples of complexity 1000. 

From this part, I cannot understand whether you meant the 100% accuracy  in training set or test set.  Or is help-out examples totally different ? I tried to get the idea from your code but it looks like a bit complicated.
